frontend/src/components/ViewActivities/index.js

```diff
@@ -92,21 +97,26 @@ class ViewActivities extends React.Component {
       if (!this.state.addingActivity) {
         return (
           <div>
+            <Button type='button' onClick={this.addActivity}>+ Add</Button>
             <div className='activity-page'> 
               <ActivityList tripId={tripId}/>
             </div>
-            <Button type='button' onClick={this.addActivity}>+ Add</Button>
           </div>
         );
       } else {
         return (
-          <Modal.Dialog>
-            <Modal.Header>Add New Activity</Modal.Header>
+          <Modal 
+          show={this.state.addingActivity}
+          onHide={this.cancelAdd}
+          dialogClassName="add-activity-modal" >
```

Comment:
Optional nit. It is most important to be consistent within your own code but I think Keiffer and I are using this syntax for element definitions like this.
```suggestion
          <Modal 
            show={this.state.addingActivity}
            onHide={this.cancelAdd}
            dialogClassName="add-activity-modal" 
          >
```